### ISSUE TITLE
Does not work with with com_adsmanager - Solved

### ISSUE THREAD

**Issue by davidetaddei** [ghazal/plg_mvcoverride#2]
Hello Ghazal, if enabled on a T3 Framework Joomla! installation generates the following error:

"Fatal error: Class 'TView' not found in /plugins/system/mvcoverride/mvcoverride.php(131) : eval()'d code on line 19"

Is there any solution?

**Comment by ghazal** [comment]
Hi,
Note : T3 templates have a very particular structure.
Lets work from the same basis, say you use latest T3 Framework plugin : 
Name: T3 Framework
Version:  2.4.5
Released Date:  Mar 05, 2015

And explain in a more detailed fashion what you did.

**Comment by davidetaddei** [comment]
Hello,
actually the problem was not with T3 but with com_adsmanager (http://extensions.joomla.org/extension/adsmanager), sorry the name TView misleaded me.

I'm using a previous version of T3, I just upgraded the plugin to 2.4.5 version. 
Joomla version is 3.4

I'm trying to extend front view on com_adsmanager component, so:

1. Plugin enabled with MAKE EXTENDABLE option ENABLED and CHANGE FUNCTIONS option DISABLED;
2. Copy t3_bs3_blank/code/com_adsmanager/views/front/view.html.php
3. Extend the class with *class AdsmanagerViewFront extends AdsmanagerViewFrontDefault()*

When loading the page I receive the same error:

Fatal error: Class 'TView' not found in /var/www/ptx.local/plugins/system/mvcoverride/mvcoverride.php(131) : eval()'d code on line 19

I've checked the code, it seems com_adsmanager uses abstract classes TController, TView and TModel classes that extend basic Joomla! classes, so this is the reason of the error.

To solve it I included the core.php file at the beginning with the TController, TView and TModel abstract classes using require_once(JPATH_SITE.'/components/com_adsmanager/lib/core.php') and everthing seems to work fine, I will test it longer in the next days.

Do we open a new issue with the correct title or modify this one?
